department-of-veterans-affairs/content-build
# 12045 skip link focus

## Description
Moves the modal element to be above the skip link in the header   
relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12045

## Testing done & Screenshots
NA


## QA steps
Open dev tools and verify that the modal component elemenet is in the header before the skip link



## Acceptance criteria

- [ ] Modal still works as expected

## Definition of done
